deploy/chart/templates/0000_50_olm_08-catalog-operator.deployment.yaml

```diff
@@ -64,26 +61,10 @@ spec:
               path: /healthz
               port: {{ .Values.catalog.service.internalPort }}
           terminationMessagePolicy: FallbackToLogsOnError
-          env:
```

Comment:
A quick note as I need to double-check against the current master branch, but this appeared to be problematic in the context of the upstream manifests as we'd essentially be rendering an empty `env` map as the only key would be that RELEASE_VERSION variable, but that conditional check would evaluate to false.